python/mlx/optimizers/optimizers.py

```diff
@@ -398,7 +398,7 @@ class Adam(Optimizer):
     r"""The Adam optimizer [1].
 
     Our Adam implementation follows the original paper and omits the bias
-    correction in the first and second moment estimates. In detail,
+    correction in the first and second moment estimates by default. In detail,
```

Comment:
I would simply remove that comment and document the `bias_correction` argument below in the args.

Reply:
I've adjusted the comment accordingly. Does this look reasonable to you?